docs/src/main/asciidoc/spanner.adoc

```diff
@@ -963,6 +964,17 @@ public String doWorkInsideTransaction() {
 
 When creating custom repositories for your own domain types and query methods, you can extend `SpannerRepository` to access Cloud Spanner-specific features as well as all features from `PagingAndSortingRepository` and `CrudRepository`.
 
+=== Query Methods
+
+`SpannerRepository` supports Query Methods.
+Described in the following sections, these are methods residing in your custom repository interfaces whose implementations are generated based on their names and annotations.
```

Comment:
Grammar: I believe "whose" should only be used to refer to people

Reply:
Yes you're technically correct, I'll change it to "of which".

Looking online it seems people generally accept "whose" for things because it sounds so much smoother than "of which". (this is similar to how using "them" as a singular pronoun is technically wrong , but it's starting to be accepted (you should use "he or she"))